lib/cmr/cmr_client.rb

```diff
@@ -74,6 +74,16 @@ def search_collections(options, token)
       get(url, options, token_header(token))
     end
 
+    def test_query(query, token, headers = {})
```

Comment:
This needs to be a post

Reply:
Turns out the CMR search API accepts up to ~500k characters, but advises the post for anything over 6k.

Reply:
After discussion with Siwei, I am inclined to leave this as a get until CMR changes their processing to a post, at which point we should change to the post.

Reply:
meaning they don't accept a POST on this interface?

Reply:
They accept a POST, but when CMR processes queries to actually send e-mails it currently uses the get end point. I think our estimate should match the endpoint CMR is using internally.

Reply:
I think you should add a comment with your conclusion